### PR TITLE
docs(guide): add example for adding user in DataHub

### PR DESCRIPTION
**Pull request by anshbansal**
## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [x] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

File: docs-website/sidebars.js [hunk 1]
```diff
@@ -184,12 +192,6 @@ module.exports = {
       // TODO: the titles of these should not be in question form in the sidebar
       "docs/developers",
       "docs/docker/development",
-      "metadata-ingestion/adding-source",
-      {
-        type: "doc",
-        label: "Ingesting files from S3",
-        id: "metadata-ingestion/source_docs/s3",
```
Comment (shirshanka) [inline docs-website/sidebars.js:191]: I think we need this one

Comment (anshbansal) [reply docs-website/sidebars.js:191]: S3 doc is present under Sources. Why do we need this specifically here?

**Comment by anshbansal** [comment]
```
sharp: Downloading https://github.com/lovell/sharp-libvips/releases/download/v8.11.3/libvips-8.11.3-linux-x64.tar.br
sharp: Installation error: Status 503 Egress is over the account limit.
```
Seems like a temporary error on one of the dependencies. Will need to re-run this.

**Review by jjoyce0510 — APPROVED** [review]
LGTM. Nice and lightweight!

**Review by shirshanka — APPROVED** [review]
LGTM

**Pull request merged**